uniflow/op/model/model_config.py

```diff
@@ -61,3 +61,15 @@ class NougatModelConfig(ModelConfig):
     model_name: str = "0.1.0-small"
     batch_size: int = 1
     model_server: str = "NougatModelServer"
+
+@dataclass
+class BedrockModelConfig(ModelConfig):
+    """Bedrock Model Config Class."""
+
+    model_name: str = "anthropic.claude-v2"
+    batch_size: int = 1
+    model_server: str = "BedrockModelServer"
+    aws_profile: str = "default"
+    aws_region: str = ""
+    # TODO: Need to consider the best approach for handling model arguments
+    model_kwargs: Dict[str, Any] = field(default_factory=lambda: {})
```

Comment:
qq: what are some other options in your mind?

Reply:
It can be challenging to create a unified template for model arguments for me at this moment, as the parameters vary for different models on Bedrock (e.g., "top_p" for Claude and "topP" for Titan). One quick solution is to select some commonly used arguments (e.g., temperature, etc.), create a unified BedrockModelConfig in model_config.py, and maintain a mapping from the unified configuration to specific models. However, this approach would impose an additional burden on the Uniflow project for maintenance. Therefore, I suggest we keep it as is and leave the choice to the user.

Ref: 
Titan: https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan-text.html
Claude: https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-claude.html